libr/bin/format/mach0/coresymbolication.c

```diff
@@ -185,13 +185,23 @@ RCoreSymCacheElement *r_coresym_cache_element_new(RBinFile *bf, RBuffer *buf, ut
 		goto beach;
 	}
 	ut8 *end = b + hdr->size;
-	if (hdr->file_name_off) {
+	if (file_name) {
+		result->file_name = file_name;
```

Comment:
use ternary operator

Reply:
i'm not sure: this would be two nested ternary operators, does it look more readable without `if`?

```c
	result->file_name = file_name?
		file_name :
		(hdr->file_name_off?
			str_dup_safe (b, b + (size_t)hdr->file_name_off, end) :
			NULL);
```